apps/meeting/package.json

```diff
@@ -11,6 +11,7 @@
     "amazon-chime-sdk-component-library-react": "^2.6.0",
     "amazon-chime-sdk-js": "^2.11.0",
     "aws-sdk": "^2.731.0",
+    "fs-extra": "^10.0.0",
```

Comment:
Where does this come from?

Reply:
This is in the [deploy.js](https://github.com/aws-samples/amazon-chime-sdk/blob/main/apps/meeting/serverless/deploy.js#L2) and we do `const fs = require('fs-extra')`. Also updated the PR description

Reply:
Probably, we missed it because the servereless package uses the `fs-extra` module defined in the root level.
https://github.com/aws/amazon-chime-sdk-js/blob/917c330a3fc6e8d2349162b00d5015f36d4edf78/package.json#L68